csiplugin/connectors/resources.go

```diff
@@ -555,8 +555,10 @@ type CreateFilesetRequest struct {
 }
 
 type CreateMakeDirRequest struct {
-	UID string `json:"uid,omitempty"`
-	GID string `json:"gid,omitempty"`
+	UID   string `json:"uid,omitempty"`
+	GID   string `json:"gid,omitempty"`
+	USER  string `json:"user,omitempty"`
```

Comment:
what is the difference between UID and USER, and GID and GROUP?  I assume one is the number and one is the name?  Perhaps a comment here would be good.

Reply:
> what is the difference between UID and USER, and GID and GROUP? I assume one is the number and one is the name? Perhaps a comment here would be good.

Agree. I have added the comments